rpcs3/rpcs3qt/skylander_dialog.h

```diff
@@ -6,7 +6,7 @@
 #include <QDialog>
 #include <QLineEdit>
 
-constexpr auto UI_SKY_NUM = 4;
+constexpr auto UI_SKY_NUM = 8;
```

Comment:
Isn't there a proper constant for this number somewhere?

Reply:
You'd have to ask the developers of the skylander games what constant they used :smirk: .